backend/endpoints/responses/search.py

```diff
@@ -6,5 +6,6 @@ class SearchRomSchema(BaseModel):
     slug: str
     name: str
     summary: str
-    url_cover: str
+    igdb_url_cover: str = ""
+    moby_url_cover: str = ""
```

Comment:
this is a good change, would be nice to have a computed field called url_cover so we don't need to do `x or y` everywhere on the client

Reply:
i'll add this computed variable in a next PR, but i totally agree with it